doc/source/whatsnew/v1.1.1.rst

```diff
@@ -16,7 +16,7 @@ Fixed regressions
 ~~~~~~~~~~~~~~~~~
 
 - Fixed regression where :func:`read_csv` would raise a ``ValueError`` when ``pandas.options.mode.use_inf_as_na`` was set to ``True`` (:issue:`35493`).
--
+- Fixed regression in :class:`pandas.core.groupby.RollingGroupby` where a segfault would occur with ``center=True`` and an odd number of values (:issue:`35552`)
```

Comment:
`pandas.core.groupby.RollingGroupby` doesn't exist, so this link won't work (it's in the `window` module). But, we actually don't have this in the API docs, so won't work anyway. I would just do something like `.groupby(..).rolling(..)` to indicate which operation is affected.